var/spack/repos/builtin/packages/lvarray/package.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import warnings
+import os
+
+
+class Lvarray(CMakePackage, CudaPackage):
```

Comment:
In LvArray I have an uberenv based build setup. This involves duplicating this `package.py` file with some additional logic to generate a CMake host config file. Following the convention of the other projects that also use uberenv (RAJA, CHAI, Umpire, Axom) I have omitted the host config logic from this package. However is this necessary? If acceptable I'd prefer to add the host config logic here so I could get rid of the very similar `package.py` in LvArray.

https://github.com/GEOSX/LvArray/blob/temp/feature/corbett/tribol/scripts/uberenv/packages/lvarray/package.py

Reply:
@corbett5 please put the host-config logic here. It’s perfectly fine from Spack perspective, and it’s what RAJA, CHAI and Umpire will do soon too).

Reply:
Done.